src/generalComponents/Selector.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const SelectorWrapper = styled.div`
+display: block;
+width: 23.1px;
+height: 19.8px;
+color: #FFF;`
+ /* display: none;
+@media (max-width: 375px) {
+    display: block; */
+
+const OptionSelector = styled.div`
+width: 23.1px;
+height: 3px;
+background: #FFF;
+color: #FFF;
+margin: 4px 0px 4px 5px;
+`
+const OpltionList = styled.ul`
+list-style: none;
+top: 30px;
+left: 0px;
+display: ${({visible})=>(visible?"block":"none")}
+`
+const SetOption = styled.li`
+cursor: pointer;
+`
+const Selector = () => {
```

Comment:
nie do końca rozumiem do czego jest właściwie ten selektor

Reply:
Selektor jesli dobrze pamietam moj tok myslenia, mial pojawiac sie w momencie wejscia app w mobilna wersje, mial byc to hamburger z mozliwoscia wyboru.